### DISCUSSION TITLE
Meta 推出全新AI 即時口譯功能!!

### DISCUSSION THREAD

**Discussion by ruthchen1020** [1101-GS4719-C/Python-Community-GS4719-C#818] [General]
台語是一個沒有文字的語言，所以製作AI翻譯機器人的難度也大大提升。
不過，meta創造了一個連口音都做到非常準確的機器人，是否意味著我們不再需要為台語失傳而擔心。
https://ynews.page.link/F3HVm

**Comment by Kotori0501** [comment]
感謝分享
身為聽不太懂台語的我，如果能利用此技術跟阿公阿嬤聊天，我覺得很好，因為可以跟長輩好好交流。如果以後可以發展不同語言，又可以和不同世界的人交流，因此我很期待！！

**Comment by evanchou1204** [comment]
之前我教阿嬤怎麼用語音輸入在youtube放歌，她竟然直接講台語，有了台語翻譯機器人，阿嬤就可以直接在youtube聽歌了！

**Comment by a45320614** [comment]
這個技術能對世界各地的文化保存帶來非常大的好處，例如台灣原住民的文化也可以透過翻譯耆老的話語快速紀錄，非常方便

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
我阿公就是個只會說台語的人 這項技術的發明對我和阿公的關係能更加緊密

**Comment by dinosaur1009** [comment]
這個發明真的很實用，對於溝通亦有幫助，希望可以越來越精準

**Comment by Zoe06060606** [comment]
透過這種方式感覺能保存更多不是實體的文化遺產

**Comment by Pietra1226** [comment]
只是不知道會不會因為可以即時翻譯導致懂得少數語言的人越來越少，大家逐漸都使用翻譯來解決溝通的問題呢?

**Comment by Evan0979** [comment]
翻譯失業要失業了嗎QQ,之後是不是都不用學外語了反正AI翻譯那麼猛

**Comment by StevenWanglolz** [comment]
竟然能即時口譯台語，太厲害了吧!不過我比較擔心的是會不會取代掉很多人的工作? 像是口譯員或是各個語言的老師?因為我們以後就不再需要學習新語言了啊，每個人都隨身攜帶一台即時翻譯器就好了

**Comment by CPCpyt** [comment]
我覺得這個超級酷，有種翻譯年糕的感覺，
希望未來能實際運用到這樣的技術。
